_current_challenges/01_workshops.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: single
+sidebar:
+  nav: workshops.md
+title: Upcoming SAMPL workshops
+excerpt: Details of upcoming SAMPL in-person and virtual workshops, including agendas as applicable.
+permalink: /current/workshops/
+---
+
+## The SAMPL6 LogP Virtual Workshop
+
+We have a virtual pre-workshop on the SAMPL6 LogP challenge on Thursday, May 19, beginning at 7 am US Pacific time.  
+
+### Overview/Purpose
+
+The purpose of this workshop is to go over a preliminary evaluation of results, begin considering analysis and lessons learned, and nucleate opportunities for follow up and additional discussion. Part of the goal is to facilitate discussion so that participants can work *together* to maximize lessons learned in the lead up to the August in-person D3R/SAMPL workshop and the JCAMD special issue.
+We hope this early workshop will result in additional follow-up afterwards.
+
+### Agenda
+
+A tentative agenda for this workshop is (all times US Pacific):
```

Comment:
I think it would help to be more clear about the time zone of the schedule here: US Pacific Time (PDT, UTC-7).